apps/recovery-portal/src/main/webapp/self-registration-complete.jsp

```diff
@@ -22,12 +22,26 @@
 <%@ page import="org.wso2.carbon.identity.mgt.endpoint.util.IdentityManagementEndpointUtil" %>
 <%@ page import="java.net.MalformedURLException" %>
 <%@ page import="java.io.File" %>
+<%@ page import="org.wso2.carbon.core.util.SignatureUtil" %>
+<%@ page import="org.owasp.encoder.Encode" %>
+<%@ page import="java.util.Base64" %>
+<%@ page import="org.wso2.carbon.identity.recovery.util.Utils" %>
+<%@ page import="java.util.HashMap" %>
+<%@ page import="java.util.List" %>
+<%@ page import="java.util.Map" %>
+<%@ page import="org.json.simple.JSONObject" %>
 
 <jsp:directive.include file="includes/localize.jsp"/>
 <%
     boolean isEmailNotificationEnabled = false;
-
+    boolean isAutoLoginEnable = false;
```

Comment:
This condition is always false right?
Why aren't we reading the config value?

Reply:
isAutoLoginEnable is checked in the line 56